umh-core/pkg/service/redpanda/redpanda.go

```diff
@@ -956,3 +962,133 @@ func formatMemory(memory int) string {
 
 	return fmt.Sprintf("%d%s", memory, units[unitIndex])
 }
+
+// updateRedpandaClusterConfig updates Redpanda's cluster config via HTTP PUT request
+func (s *RedpandaService) UpdateRedpandaClusterConfig(ctx context.Context, redpandaName string, configUpdates map[string]interface{}) error {
+	if s.httpClient == nil {
+		return fmt.Errorf("http client not initialized")
+	}
+
+	// If the context is done, return an error
+	if ctx.Err() != nil {
```

Comment:
maybe it makes sense to define our own constant for "UpdateHTTPTimeout" or something similar so that we tell future developers that this only takes like xx ms maximum. helps to spot regressions as well